Recipes.md

```diff
@@ -372,3 +372,15 @@ run({
   saveAs: "png"
 });
 ```
+
```

Comment:
If we would extract path from `homepage` we can remove this documentation - one thing less to think for the user.

Reply:
Yep cool 👍 i will make that change